File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/recyclerView/selector/MeteoriteSelector.java
package com.antonio.samir.meteoritelandingsspots.ui.recyclerView.selector;


import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

/**
 * Meteorite Selector interface to abstract the selection command from Meteorite Adapter
 */
public interface MeteoriteSelector {

    void selectItemId(final String meteorite);

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/features/list/ui/recyclerView/selector/MeteoriteSelector.kt
package com.antonio.samir.meteoritelandingsspots.features.list.ui.recyclerView.selector

import com.antonio.samir.meteoritelandingsspots.data.repository.model.Meteorite


/**
 * Meteorite Selector interface to abstract the selection command from Meteorite Adapter
 */
interface MeteoriteSelector {

    fun selectItem(meteorite: Meteorite)

}
